tests/unit/test_textUtils.py

```diff
@@ -0,0 +1,227 @@
+# -*- coding: UTF-8 -*-
+#tests/unit/test_textUtils.py
+#A part of NonVisual Desktop Access (NVDA)
+#This file is covered by the GNU General Public License.
+#See the file COPYING for more details.
+#Copyright (C) 2019 NV Access Limited, Babbage B.V., Leonard de Ruijter
+
+"""Unit tests for the textUtils module."""
+
+import unittest
+from textUtils import WideStringOffsetConverter
+
+class TestStrToWideOffsets(unittest.TestCase):
+	"""
+	Tests that ensure that offsets in a string are properly converted to wide string offsets.
+	Every string offset for 32-bit unicode characters (e.g. emoji) take two offsets in a wide string representation.
+	"""
+
+	def test_nonSurrogate(self):
+		converter = WideStringOffsetConverter(text="abc")
+		self.assertEqual(converter.wideStringLength, 3)
```

Comment:
This really looks like a perfect time to write data driven tests, though I think we would need to use `nose` or `pytest` to get nice syntax for that.